src/pages/set-password.tsx

```diff
@@ -93,14 +93,14 @@ export const SetPasswordPage: React.FC<SetPasswordProps> = ({
   });
 
   return (
-    <Formik
-      initialValues={{ password: '' }}
-      validationSchema={validationSchema}
-      onSubmit={handleSubmit}
-    >
-      {formik => (
-        <Form>
-          <PopupContainer header={<Header hideActions title="Set a password" />} requestType="auth">
+    <PopupContainer header={<Header hideActions title="Set a password" />} requestType="auth">
```

Comment:
It looks like the root cause is that `PopupContainer` was nested in the form, which caused global styles to apply incorrectly

Reply:
yep, that makes sense, the form should not have had the container nested in it.